frontend/staff-dashboard/src/pages/flexible_pricing/list.tsx

```diff
@@ -98,6 +73,8 @@ const FlexiblePricingFilterForm: React.FC<{ formProps: FormProps }> = ({ formPro
 }
 
 export const FlexiblePricingList: React.FC = () => {
+    const { open: displayToast } = useNotification();
```

Comment:
Is this still being used?

Reply:
Fixed in https://github.com/mitodl/mitxonline/pull/668/commits/eaee7b46d3a1e30a049498afbefa1d8201a8319d